server/middlewares/audioParser.js

```diff
@@ -0,0 +1,22 @@
+import multer from 'multer';
+import cloudinary from 'cloudinary';
+import cloudinaryStorage from 'multer-storage-cloudinary';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+cloudinary.config({
+    api_key: process.env.API_KEY,
+    api_secret: process.env.API_SECRET,
+    cloud_name: process.env.CLOUD_NAME,
+});
+const storage = cloudinaryStorage({
+    cloudinary,
+    params: {
+        folder: 'audio',
+        format: 'mp3',
+        resource_type: 'video',
+    },
+});
+
+export default multer({ storage });
```

Comment:
this file is exactly the same as `imageParser.js` expect for the `formate`. Create a wrapper function where you pass in the appropriate `format` instead of duplicating the code